File: t/VB/MedicalViewerLayoutDemo/UI/DefaultImagesDialog.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace MedicalViewerLayoutDemo
   Public Partial Class DefaultImagesDialog : Inherits Form
	  Public Sub New()
		 InitializeComponent()
	  End Sub

	  Private Sub okButton_Click(ByVal sender As Object, ByVal e As EventArgs) Handles okButton.Click
		 If loadRadio.Checked Then
			 Me.DialogResult = DialogResult.Yes
		 Else
			 Me.DialogResult = DialogResult.No
		 End If
		 Me.Close()
	  End Sub
   End Class
End Namespace

File: t/CS/MedicalViewerDemo/UI/DefaultImagesDialog.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace MedicalViewerDemo
{
   public partial class DefaultImagesDialog : Form
   {
      public DefaultImagesDialog()
      {
         InitializeComponent();
      }

      private void okButton_Click(object sender, EventArgs e)
      {
         this.DialogResult = loadRadio.Checked ? DialogResult.Yes : DialogResult.No;
         this.Close();
      }
   }
}
